autopilot/migrator.go

```diff
@@ -130,6 +130,12 @@ func (m *migrator) performMigrations(p *workerPool, set string) {
 
 OUTER:
 	for {
+		// recompute health first.
+		if err := b.RecomputeHealth(ctx); err != nil {
```

Comment:
I think it makes sense to add some logging here to indicate the duration of the computation so we can keep an eye on it in the logs.